packages/contracts/src/manager/libraries/LibAppStorage.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
elsewhere it is 0.8.8 can you push all of them to 0.8.16 ?

Reply:
> elsewhere it is 0.8.8 can you push all of them to 0.8.16 ?

That pragma version was from solidstate-solidity, which has been audited by Hacken. I didn't want to generate any vulnerability by changing version